docs/Doku_PlanBuilder.md

```diff
@@ -0,0 +1,163 @@
+Documentation Plan Builder
+
+The OpenTOSCA-Container uses the Tosca Runtime to generate build and termination plans, in BPEL format, out of given CSAR-Files.
```

Comment:
`OpenTOSCA Container`. No dashes in English words.

The OpenTOSCA Container is a TOSCA Runtime.
It consists of a build plan generator, which can generate plans for building and terminating applications.
It takes a topology template as input.
It uses BPEL as workflow language.